metriq_gym/benchmarks/clops.py

```diff
@@ -36,4 +38,24 @@ def dispatch_handler(self) -> None:
         self.job_manager.add_job(partial_result.to_serializable())
 
     def poll_handler(self) -> None:
-        pass
+        logging.info(f"Polling results for CLOPS job with job-id {self.args.job_id} results.")
+        results = poll_job_results(self.args.jobs_file, BenchJobType.CLOPS)
```

Comment:
Perhaps that's what you meant in your PR comment, but isn't this going to pull _all_ CLOPS jobs? Do we want this already? Is it to keep the PR small?

Reply:
The internals of the `poll_job_results` function does indeed poll by ID, however I've carelessly left `BenchJobType.CLOPS` in the second argument. I changed that to `self.args.job_id`, which now, takes this, and properly polls *only* the jobs in the `.*jsonl` file that match with that metriq-gym job ID. 

The same was done for quantum volume as well. And, since the polling of results is identical, this now lives in the base `Benchmark` class. We can pull this back into the subclasses if this differs for any specific reason, but the only difference I anticipate is a difference with how this is done w.r.t. different providers (which I think would be appropriately dealt with in the base `Benchmark` class as well).

Done in df55ef3.